src/components/sidebar/SafeList/index.tsx

```diff
@@ -59,13 +64,55 @@ export const _shouldExpandSafeList = ({
 const MAX_EXPANDED_SAFES = 3
 const NO_SAFE_MESSAGE = 'Create a new safe or add'
 
+export const addActionsToTxs = (
+  chainId: string,
+  address: string,
+  page: TransactionListPage,
+  isSafeOwned: boolean,
+  txsActionsByChain: Record<string, Record<string, SafeActions>>,
+  walletAddress?: string,
```

Comment:
Nit: can you pass these arguments as an object. It's hard to follow them when called.